ci/build_alpine_container/README.md

```diff
@@ -0,0 +1,17 @@
+This directory contains a Dockerfile that builds an envoy image
```

Comment:
@mattklein123 Roger, thanks for clarifying on that. Referencing its Dockerfile, this image bakes in glibc 2.25 - https://hub.docker.com/r/frolvlad/alpine-glibc/~/dockerfile/.